tap_marketo/__init__.py

```diff
@@ -299,28 +303,28 @@ def do_sync():
     # the leads.
     for activity_type_id in LEADS_CHANGED_IDS:
         activity_type_ids.remove(activity_type_id)
-        logger.info("Syncing lead-altering activity type %d", activity_type_id)
+        LOGGER.info("Syncing lead-altering activity type %d", activity_type_id)
         sync_activities(activity_type_id, lead_fields, date_fields, leads_schema, do_leads=True)
 
     # If there are any unsynced leads after the last mutating activity type,
     # sync them before continuing
-    if len(LEAD_IDS) > 0:
+    if LEAD_IDS > 0:
```

Comment:
This change does make sense. What does it mean to compare a set of things with the number 0?